acl/types.go

```diff
@@ -1,5 +1,17 @@
 package acl
 
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+type BasicACL uint32
+
+func (a BasicACL) String() string {
+	return fmt.Sprintf("0x%x", uint32(a))
```

Comment:
Maybe it should return human readable names there as well, such as `public-read` there? 
What do you think @fyrchik @carpawell ?

Reply:
I think it is better have consistent format. If we add yet another "known" rule, we don't want to break any code dependent on string format.

Reply:
I also think, it should be padded, i.e. `%08x`. It is a bit-mask with fixed length.